crates/turborepo-ui/src/tui/app.rs

```diff
@@ -77,6 +77,10 @@ impl<I> App<I> {
     pub fn term_size(&self) -> (u16, u16) {
         self.pane.term_size()
     }
+
+    pub fn update_tasks(&mut self, tasks: Vec<String>) {
+        self.table = TaskTable::new(tasks.clone());
```

Comment:
We need to be clearing and updating the panes in some way as well. Right now they get reused so task output from a previous run is still displayed.

Reply:
Is that bad? I kind of like seeing the previous runs, so you can understand that something got re-run. If a task completes too quickly, it could end up showing the same output which would confuse people